docs/sources/helm-charts/mimir-distributed/migration-guides/migrate-enable-cluster-label-verification/_index.md

```diff
@@ -0,0 +1,96 @@
+---
+description: "Migrating Mimir Helm Installation to Use Memberlist Cluster Label"
```

Comment:
I think the description should provide more information on what the reader can expect to get from this page.
The description is presented to the user in search engines and on social media when links to this page are used.

A good example is https://github.com/grafana/mimir/blob/main/docs/sources/mimir/set-up/migrate/migrate-from-cortex.md?plain=1#L4-L7

https://grafana.com/docs/writers-toolkit/write/front-matter/#description-required